packages/jest-console/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased (2.0.0)
+
+- Add new API methods `toHaveInformed`, `toHaveInformedWith`, `toHaveLogged` and `toHaveLoggedWith` ([137](https://github.com/WordPress/packages/pull/137))
```

Comment:
We should clarify that this is a breaking change ("Breaking:" prefix) and some hint to in what way it breaks / how consumers can update their code.

Reply:
Good point, I will add details.